cmd/helm/helm.go

```diff
@@ -45,7 +46,8 @@ func init() {
 
 func debug(format string, v ...interface{}) {
 	if settings.Debug {
-		format = fmt.Sprintf("[debug] %s\n", format)
+		timeNow := time.Now().UTC().String()
```

Comment:
Should the default timezone be UTC or the timezone the machine is set in?

If I'm using Helm on my local system and the time is UTC (I'm not in UTC) it will look funny. If someone is in India, they are off by 30 minutes in addition to hours. The time displayed may be confusing.

I would suggest using the local time as the default. If you want UTC that should be opt-in (e.g., with an environment variable).

Reply:
good point! thanks @mattfarina